extensions/amp-story-panning-media/README.md

```diff
@@ -0,0 +1,42 @@
+# Interactive components
+
+## Summary
+
+Use `amp-story-panning-media` to transition an image between pages.  
+This enables building stories around a single image and creating page transition effects like parallax.
+
+A more complete version of this document can be found in [amp.dev](https://amp.dev/documentation/components/amp-story-panning-media/?format=stories).
+
+## Demos
+
+<table>
+  <tr>
+   <td><strong>The Northern Sky Constellations:</strong> This story uses a star chart for the entire story by panning and zooming into different constellations. A parallax transition effect is also achieved by layering two png star images with transparency.
+<br><br>
+<a href="https://philipbell-panning-media.web.app/examples/amp-story/northern-sky-constellations.html">The Northern Sky Constellations</a>
+   </td>
+<td>
+<img src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/constellations-screenshot.jpg">
+   </td>
+  </tr>
+<tr>
+<td>
+<img src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/controls.jpg">
+   </td>
+   <td><strong>Web Story creation tool integration example:</strong>Use the sliders to modify the attributes of the panning-media component on the current page.
+<br><br>
+<a href="https://philipbell-panning-media.web.app/examples/amp-story/controls.html">https://webstoriesinteractivity-beta.web.app/animals-polls.html</a>
+   </td>
+</tr>
+</table>
+
+## FAQ
+
+**What's next for `amp-story-panning-media`?**
```

Comment:
If linking to the roadmap is important, you can move this information into a [tip](https://amp.dev/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting/?format=websites#tips) inside the reference documentation. My instinct says it would fit best after the first paragraph under Usage or under Validation.